src/pages/Gallery/Gallery.tsx

```diff
@@ -1,7 +1,118 @@
-import React from "react"
+import React, { useState } from "react"
+
+import styles from "./Gallery.module.scss"
+import classnames from "classnames"
+
+const Lines = () => {
+  return (
+    <div className={styles.lines}>
+      {Array.from({ length: 6 }).map((_, index) => (
+        <div key={index} className={styles.line} />
+      ))}
+    </div>
+  )
+}
```

Comment:
Okay, we are using this on multiple places so let's extract it with it's css and put there into its own component in `components` folder